frontend/src/components/pages/HomePage.tsx

```diff
@@ -78,6 +99,11 @@ const HomePage = () => {
           />
         </Flex>
       </Flex>
+      {showScrollToTop && (
+        <Button onClick={handleScrollToTop} size="lg" variant="scrollToTop">
+          <Icon as={MdKeyboardArrowUp} h={8} w={8} />
```

Comment:
nit: IMO chakra is easier to read when attributes are not short form, ie use `height` and `width`